### ISSUE TITLE
Can't upload file to custom file input.

### ISSUE THREAD

**Issue by viengiang** [abramenal/cypress-file-upload#266]
### Current behavior:

Thanks for your work, this helps me a lot in e2e-testing.
When I tried to upload a file to a custom file input, I don't see anything happen. 
![image](https://user-images.githubusercontent.com/61610632/105462568-5e91f780-5cc1-11eb-9d08-559f1291ea2e.png)

![image](https://user-images.githubusercontent.com/61610632/105462426-2f7b8600-5cc1-11eb-8c24-3baaa2ce5b11.png)
So I change the CSS and try again
![image](https://user-images.githubusercontent.com/61610632/105463088-2212cb80-5cc2-11eb-9b51-9045ad5bf939.png)
But the result doesn't like my expectations.
![image](https://user-images.githubusercontent.com/61610632/105463107-2939d980-5cc2-11eb-827e-14f8cb1e384f.png)

### Desired behavior:
My expectations are:
When I upload the correct file:
![image](https://user-images.githubusercontent.com/61610632/105463444-a402f480-5cc2-11eb-8a35-7e16c7c0a7a8.png)
![image](https://user-images.githubusercontent.com/61610632/105463451-a7967b80-5cc2-11eb-8f32-2063cc3c0252.png)

When I upload the incorrect file:
![image](https://user-images.githubusercontent.com/61610632/105463335-7fa71800-5cc2-11eb-96d9-1fda21383e1b.png)

### Steps to reproduce: (app code and test code)
Here is my code, can anybody help my, please? Many thanks.
`cy.get(MY_CONTENT.cssSelector.availableTypes.cmp_product_catalog.cmp_upload_pdf.btn_add_pdf_file)
    .attachFile(pdf_file_20mb, {force: true});`

**Comment by abramenal** [comment]
Hi @viengiang 
Thanks for submitting the issue!

That is great. I think what you're missing is correct events to be handled.
So, as you see, the file is actually gets attached to the `input#fileInput` element. I think the difference you're facing when attaching file manually vs doing it with the plugin is events.

I suggest to check out your implementation and see, on which events do you trigger your logic of validating/uploading a file.
You might find one of these events used: https://github.com/abramenal/cypress-file-upload/blob/main/src/constants.js#L14
If it is so, I suggest to change your command to `cy.get().attachFile(pdf_file_20mb, { subjectType: 'drag-n-drop' })`

**Comment by viengiang** [comment]
Thanks for your sharing @abramenal 

After reading the comment, I already tried to find out the event and the "click" event is what I need. So, I tried to click first and attach the file after. This actually works.

`cy.get(MY_CONTENT.cssSelector.availableTypes.cmp_product_catalog.cmp_upload_pdf.btn_add_pdf_file).click().attachFile(pdf_file_20mb, { force: true });`

Thank you again. Have a great day :)
